server/app/controllers/applicant/RedirectController.java

```diff
@@ -152,30 +160,83 @@ public CompletionStage<Result> considerRegister(
           badRequest("You are not signed in - you cannot perform this action."));
     }
 
-    CompletionStage<Optional<String>> applicantName = applicantService.getName(applicantId);
-    CompletionStage<ReadOnlyApplicantProgramService> roApplicantProgramServiceCompletionStage =
-        applicantService.getReadOnlyApplicantProgramService(applicantId, programId);
-
-    return applicantName
+    CompletableFuture<Boolean> isCommonIntake =
```

Comment:
**suggestion (non-blocking):** consider adding at least a happy-path unit test for this controller action

Reply:
I created #4590 to track this